### DISCUSSION TITLE
How to create a Docker image with SBOM attestation?

### DISCUSSION THREAD

**Discussion by mendorf** [bmuschko/gradle-docker-plugin#1218] [Q&A] [answered]
Hi there! Does someone have a starting point for me or some instructions on how I would be able to use the latest docker build attestation features? 
e.g. the Docker blog has some [article](https://www.docker.com/blog/generate-sboms-with-buildkit/) about this.

**Comment by bmuschko** [comment, marked as answer]
The plugin only uses the Docker remote API, which at this time does not buildkit functionality. Therefore, SBOM generation is not available. You want to look into other plugins that call buildkit directly.
